spec_integration/queue_runner.rb

```diff
@@ -0,0 +1,75 @@
+require 'knapsack_pro'
+require 'json'
+
+ENV['KNAPSACK_PRO_CI_NODE_BUILD_ID'] = SecureRandom.uuid
+ENV['KNAPSACK_PRO_TEST_SUITE_TOKEN_RSPEC'] = 'fake-api-token'
```

Comment:
I'd go with something that makes it clear `fake-api-token` does not have a meaning in the codebase:

```suggestion
ENV['KNAPSACK_PRO_TEST_SUITE_TOKEN_RSPEC'] = SecureRandom.alphanumeric
```